infoschema/tables.go

```diff
@@ -1289,6 +1298,35 @@ var tablePlacementPolicyCols = []columnInfo{
 	{name: "CONSTRAINTS", tp: mysql.TypeVarchar, size: 1024},
 }
 
+var tableClientErrorsSummaryGlobalCols = []columnInfo{
+	{name: "ERROR_NUMBER", tp: mysql.TypeLonglong, size: 64, flag: mysql.NotNullFlag},
+	{name: "ERROR_MESSAGE", tp: mysql.TypeVarchar, size: 1024, flag: mysql.NotNullFlag},
+	{name: "ERROR_COUNT", tp: mysql.TypeLonglong, size: 64, flag: mysql.NotNullFlag},
+	{name: "WARNING_COUNT", tp: mysql.TypeLonglong, size: 64, flag: mysql.NotNullFlag},
+	{name: "FIRST_SEEN", tp: mysql.TypeTimestamp, size: 26},
```

Comment:
Not related to this PR though, but I'd like to talk a little bit about the `Timestamp`:

As mentioned by https://dev.mysql.com/doc/refman/8.0/en/datetime.html:

> TIMESTAMP has a range of '1970-01-01 00:00:01' UTC to '2038-01-19 03:14:07' UTC.

Do you think we need to consider this limitation for now?

Reply:
Yes, I think it will become critical soon. We are only 17 years away from Y2K38. The issue to track is: https://github.com/pingcap/tidb/issues/20522